packages/react-native-web/src/exports/CheckBox/index.js

```diff
@@ -26,49 +26,51 @@ type CheckBoxProps = {
   value?: boolean
 };
 
-const CheckBox = forwardRef<CheckBoxProps, *>((props, forwardedRef) => {
-  const { color, disabled, onChange, onValueChange, style, value, ...other } = props;
+const CheckBox: React$AbstractComponent<CheckBoxProps, mixed> = forwardRef<CheckBoxProps, *>(
```

Comment:
Why does this component use `mixed` instead of an explicit type?

Reply:
Mixed is the equivalent of *. Updated these to use explicit types.